index.js

```diff
@@ -0,0 +1,143 @@
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips/';
+
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p><ul>`;
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+
+// LOAD ALL TRIPS
+
+
+const loadTrips = () => {
+  reportStatus('Loading trips...');
+
+  const tripsList = $('#trips-list');
+  tripsList.empty();
+
+  axios.get(URL)
+  .then((response) => {
+    console.log(response);
+    reportStatus(`Successfully loaded ${response.data.length} pets`);
+    response.data.forEach((trip) => {
+      tripsList.append(`<li id="${trip.id}">${trip.name.toUpperCase()}</li>`);
+    });
+  })
+  .catch((error) => {
+    reportStatus(`Encountered an error while loading pets: ${error.message}`);
+    console.log(error);
+  });
+};
+
+
+//LOAD ONE TRIP
+
+
+
+const loadOneTrip = (event) => {
+  console.log(event.target.id);
+  const tripid = event.target.id;
+  reportStatus("Loading trip...");
+  const trip = $('#trip');
+  trip.empty();
+  const reservation = $(`#reserve-form`);
+  reservation.empty();
+  const tripTitle = $(`#triptitle`);
+  tripTitle.empty();
+
+  axios.get(URL+tripid)
+  .then((response) => {
+    console.log(response);
+    reportStatus(`Successfully loaded trip to: ${response.data.name}`);
+
+    trip.append(
+      `<tr><th>Duration</th><td>${response.data.weeks}</td></tr>
+      <tr><th>ID</th><td>${response.data.id}</td></tr>
+      <tr><th>Category</th><td>${response.data.category}</td></tr>
+      <tr><th>Continent</th><td>${response.data.continent}</td></tr>
+      <tr><th>About</th><td><div id="about">${response.data.about}</div></td></tr>
```

Comment:
Putting IDs on this field and the cost field create issues with your HTML - IDs are reserved for single HTML elements and this will result in multiple HTML elements having the same ID